api/src/decorators/shapefile.decorator.ts

```diff
@@ -8,6 +8,26 @@ import {
 
 import { ShapefileGeoJSONResponseDTO } from 'modules/scenarios/dto/shapefile.geojson.response.dto';
 
+export function ApiConsumesShape() {
```

Comment:
If the only current and expected difference from `@ApiConsumesShapefile()` is that there would be no geojson in the response here, could we make this a parameter, or change `@ApiConsumesShapefile()` to be a combination of this other decorator and `@ApiOkResponse()`?

Either way, if we want to have two distinct decorators, we should clarify their name a bit or someone looking at them (even ourselves when applying them to new endpoints) will need to dig into the inner `applyDecorators()` to understand what is the difference.

Reply:
Adjusted in https://github.com/Vizzuality/marxan-cloud/pull/174/commits/2a3b01804529a4fd1af49ecc3a703616d8547247